exporter/dorisexporter/exporter_logs_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package dorisexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/dorisexporter"
+
+import (
+	"context"
+	"net/http"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+	"go.opentelemetry.io/collector/pdata/plog"
+	semconv "go.opentelemetry.io/collector/semconv/v1.25.0"
+)
+
+func TestPushLogData(t *testing.T) {
+	config := createDefaultConfig().(*Config)
+	config.Endpoint = "http://127.0.0.1:18031"
+	config.CreateSchema = false
+
+	err := config.Validate()
+	require.NoError(t, err)
+
+	exporter := newLogsExporter(nil, config, testTelemetrySettings)
+
+	ctx := context.Background()
+
+	client, err := createDorisHTTPClient(ctx, config, nil, testTelemetrySettings)
+	require.NoError(t, err)
+	require.NotNil(t, client)
+
+	exporter.client = client
+
+	defer func() {
+		_ = exporter.shutdown(ctx)
+	}()
+
+	server := &http.Server{
+		ReadTimeout: 5 * time.Second,
+	}
+	go func() {
+		http.HandleFunc("/api/otel/otel_logs/_stream_load", func(w http.ResponseWriter, _ *http.Request) {
+			w.WriteHeader(http.StatusOK)
+			_, _ = w.Write([]byte(`{"Status":"Success"}`))
+		})
+		server.Addr = ":18031"
+		_ = server.ListenAndServe()
```

Comment:
Can you create the listener first and make it listen on port 0, to map to a free port, and then bind to the server 
```
l, err := net.Listen("tcp", "localhost:0")
```
then use with the server:
```
server.Serve(l, handler)
```